### ISSUE TITLE
 AttributeError: module 'keras.backend.tensorflow_backend' has no attribute '_SESSION'

### ISSUE THREAD

**Issue by clstaudt** [shap/shap#976]
First steps with SHAP on a simple MNIST classifier network:

```
net = keras.models.Sequential([
    keras.layers.Input(shape=(28,28)),
    keras.layers.Flatten(),
    keras.layers.Dense(128, activation="relu"),
    keras.layers.Dropout(0.2),
    keras.layers.Dense(n_classes, activation="softmax")
])
net.compile(
    optimizer='adam',
    loss='sparse_categorical_crossentropy',
    metrics=['accuracy']
)
net.fit(
    X_train, 
    y_train, 
    epochs=10, 
)

background = X_train[numpy.random.choice(X_train.shape[0], 100, replace=False)]

shap.DeepExplainer(net, background)
```

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-17-f3329c8e4398> in <module>
----> 1 shap.DeepExplainer(net, background)

~/anaconda3/lib/python3.7/site-packages/shap/explainers/deep/__init__.py in __init__(self, model, data, session, learning_phase_flags)
     78 
     79         if framework == 'tensorflow':
---> 80             self.explainer = TFDeepExplainer(model, data, session, learning_phase_flags)
     81         elif framework == 'pytorch':
     82             self.explainer = PyTorchDeepExplainer(model, data)

~/anaconda3/lib/python3.7/site-packages/shap/explainers/deep/deep_tf.py in __init__(self, model, data, session, learning_phase_flags)
    113         if session is None:
    114             # if keras is installed and already has a session then use it
--> 115             if keras is not None and keras.backend.tensorflow_backend._SESSION is not None:
    116                 session = keras.backend.get_session()
    117             else:

AttributeError: module 'keras.backend.tensorflow_backend' has no attribute '_SESSION'
```

**Comment by slundberg** [comment]
This is a TF 2.0 issue and should get fixed by #975 :)

**Comment by Luk164** [comment]
Just hit the same issue, any workaround yet?
